PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -594,6 +594,8 @@ def _ast_names(names):
             result.append(nm.name)
         else:
             result.append(nm)
+
+    result = [r for r in result if r != '__main__']
```

Comment:
Could you please create a pull request for the modulegraph library where ignoring the __main__ could be discussed with the modulgraph author?

https://bitbucket.org/ronaldoussoren/modulegraph